pkg/volume/csi/csi_client.go

```diff
@@ -111,6 +119,12 @@ const (
 	steps           = 5
 )
 
+type usageCount struct {
```

Comment:
I am new to K8s community, just out of my curiosity. What are the reasons to introduce a new type here instead of re-use volume.Metrics type? it needs to eventually translated later anyways. Also, is unit testing module required in this particular repo? thanks

Reply:
Basically, some drivers return only `BYTES` and some  returns only `INODES`.  A clear separation on these `index` is the main reason. Also, with usageCount driver can set these fields in `int64`.

Reply:
thinking bit back, I think agree with @yuxiangqian. We are doing same `switch..case` in two places which seems wasteful.

Reply:
Sure, if more than one vote, it has to be accommodated :) , used volume.Metrics struct in the latest patch set.